app/presenters/sufia/file_set_presenter.rb

```diff
@@ -6,6 +6,14 @@ class FileSetPresenter < ::CurationConcerns::FileSetPresenter
     delegate :depositor, :keyword, :date_created, :date_modified, :itemtype,
              to: :solr_document
 
+    def page_title
+      title.first
+    end
+
+    def link_name
+      current_ability.can?(:read, id) ? page_title : 'File'
+    end
+
```

Comment:
Haha.

I'm saying, "this def needs some unit tests in a presenter spec, *and* I think @awead is asking for integration spec coverage too, in which case I'm :+1: on adding a line to an existing feature spec and :-1: on using view specs for integration testing" though, as always, I am happy to hear counterargs!